server/routes/caseNotes/confirmation/newCaseNoteConfirmationPresenter.ts

```diff
@@ -0,0 +1,41 @@
+import { SummaryListItem } from '../../../utils/summaryList'
+import PresenterUtils from '../../../utils/presenterUtils'
+import utils from '../../../utils/utils'
+import SentReferral from '../../../models/sentReferral'
+import Intervention from '../../../models/intervention'
+
+export default class NewCaseNoteConfirmationPresenter {
```

Comment:
again, could be an idea to rename this to `AddCaseNoteConfirmationPresenter` like the others

Reply:
renamed